index.html

```diff
@@ -198,158 +165,249 @@ <h2>Tonic</h2>
+                <button type="button" class="click-btn">See project</button>
               </section>
-            </li>
-          </ul>
-        </div>
-      </section>
-      <section class="about-me">
-        <div class="about-me-holder">
-          <section>
-            <h2 class="head">About Myself</h2>
-            <p class="helloo">
-              Hello I’m a software developer! I can help you build a product ,
-              feature or website Look through some of my work and experience! If
-              you like what you see and have a project you need coded, don’t
-              hestiate to contact me.
-            </p>
-            <h4 class="ico">LET’S CONNECT</h4>
-            <ul class="ico-img">
-              <li>
-                <a href="#"
-                  ><img
-                    class="t-social-media"
-                    src="./images/twitter.png"
-                    alt="twitter"
-                /></a>
-              </li>
-              <li>
-                <a href="#"
-                  ><img
-                    class="l-social-media"
-                    src="./images/linkedin.png"
-                    alt="linkedIn"
-                /></a>
-              </li>
-              <li>
-                <a href="#"
-                  ><img
-                    class="m-social-media"
-                    src="./images/medium.png"
-                    alt="medium"
-                /></a>
-              </li>
-              <li>
-                <a href="#"
-                  ><img
-                    class="g-social-media"
-                    src="./images/github.png"
-                    alt="github"
-                /></a>
-              </li>
-              <li>
-                <a href="#"
-                  ><img
-                    class="v-social-media"
-                    src="./images/vec.png"
-                    alt="vector"
-                /></a>
-              </li>
-            </ul>
-            <button type="button" class="resume-btn">Get my resume</button>
-          </section>
-          <section>
-            <ul class="lang">
-              <li class="lang-exp">
-                <div class="no-1">
-                  <p>Languages</p>
-                  <img src="./images/arrows-down.png" alt="arrow_bottom" />
+            </div>
+          </li>
+        </ul>
+      </div>
+
+      <div class="box-contain">
+        <ul class="box-list">
+          <li class="box-item item3">
+            <img
+              class="box-image"
+              src="./images/Snapshoot-Portfolio.png"
+              alt="Portfolio Portfolio"
+            />
+            <section>
+              <div class="lists-container">
+                <h2>Facebook 360</h2>
+                <div class="lists">
+                  <span class="lists-item obect1">CANOPY</span>
+                  <img
+                    class="btno"
+                    src="images/Counter.png"
```

Comment:
- Please work on the proportion of the contents in the project card section, so the image and the description of the project are properly aligned as in the Figma design.

##### Your design
![Screenshot from 2022-07-16 10-14-31](https://user-images.githubusercontent.com/58771507/179348721-2ca7a968-7641-4a64-abb9-f87cc36c0ab6.png)

##### Figma design
![Screenshot from 2022-07-16 10-15-37](https://user-images.githubusercontent.com/58771507/179348756-75cc1448-0489-466a-941f-0ed00c27df36.png)